memor/session.py

```diff
@@ -154,16 +154,67 @@ def add_message(self,
             self._messages_status.insert(index, status)
         self._date_modified = get_time_utc()
 
-    def remove_message(self, index: int) -> None:
+    def get_message_by_index(self, index: Union[int, slice]) -> Union[Prompt, Response]:
         """
-        Remove a message from the session object.
+        Get a message from the session object by index.
```

Comment:
same

Reply:
fix in https://github.com/openscilab/memor/pull/108/commits/c53c4827ce9608a0918dfa7b5b583b76a9332918